<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuantumGui</class>
 <widget class="QDialog" name="QuantumGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>581</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Vital Stats</string>
   </property>
   <widget class="QLabel" name="labelErrorCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Error Code:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelVoltage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Voltage:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Internal Temperature:</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelBandStatus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Band Status:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWavelength">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Center Wavelength:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWingshift">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Wingshift:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLockButtons">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lock Buttons</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonDown">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonUp">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonCenter">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>541</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
